src/components/Layout/Layout/index.tsx

```diff
@@ -68,7 +68,7 @@ const Layout: React.FC<Props> = ({
 
         {children}
       </div>
-      <Footer isContactPage={isContactPage} isWhiteBg={footerWhiteBg} />
+      {/* <Footer isContactPage={isContactPage} isWhiteBg={footerWhiteBg} /> */}
```

Comment:
Why Footer is hidden?

Reply:
I was just checking something, but the Footer should be visible.